helm-chart/ray-cluster/templates/raycluster-cluster.yaml

```diff
@@ -14,6 +14,17 @@ spec:
   {{- if .Values.head.enableInTreeAutoscaling }}
   enableInTreeAutoscaling: {{ .Values.head.enableInTreeAutoscaling }}
   {{- end }}
+  {{- if .Values.head.autoscalerOptions }}
+  {{- $values := .Values.head.autoscalerOptions}}
+  autoscalerOptions: 
+    upscalingMode: {{ $values.upscalingMode | default "Default" }}
```

Comment:
All of these fields are optional -- It's better not to set any defaults here. Let's leave decisions on default to the KubeRay operator.